src/rdposix.h

```diff
@@ -108,12 +126,22 @@
  */
 static RD_INLINE RD_UNUSED
 void rd_usleep (int usec, rd_atomic32_t *terminate) {
+#ifdef __370__
+    if(usec > 1000000)
+    {
```

Comment:
ok fixing

wrt to open-brace.. yeah sorry.. finger memory is hard to fight sometimes :-)